File: source/NetLayer/NetMessage.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Threading;

namespace NetLayer
{
    /// <summary>
    /// Clase que representa un mensaje recibido o enviado por la RED
    /// </summary>
    public class NetMessage
    {
        /// <summary>
        /// El cuerpo del mensaje
        /// </summary>
        private byte[] _body;

        /// <summary>
        /// Constructor
        /// </summary>
        /// <param name="body">El contenido del mensaje</param>
        public NetMessage(byte[] body)
        {
            Body = body;
        }
        
        /// <summary>
        /// El contenido del mensaje
        /// </summary>
        public byte[] Body
        {
            get { return _body; }
            set { _body = value; }
        }

        /// <summary>
        /// El largo de este mensaje, en número de bytes
        /// </summary>
        /// <returns></returns>
        public Int32 getSize()
        {
            return Body.Length;
        }
    }
}

File: source/java/hlmp/NetLayer/NetMessage.java
package hlmp.NetLayer;

import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
/**
 * Clase que representa un mensaje recibido o enviado por la RED
 */
public class NetMessage {
	
	/**
	 * El cuerpo del mensaje
	 */
	private byte[] body;
	
	public NetMessage(){
		this.body = new byte[1];
		body[0]=0;
	}
	
	/**
	 * Constructor
	 * @param body El contenido del mensaje
	 */
	public NetMessage(byte[] body) {
		super();
		this.body = body;
	}

	public byte[] getBody() {
		return this.body;
	}

	public void setBody(byte[] body) {
		this.body = body;
	}


	/**
	 * El largo de este mensaje, en número de bytes
	 * @return
	 */
	public int getSize(){
		return this.body.length;
	}
}